app/models/client.rb

```diff
@@ -10,11 +10,28 @@ class Client < ApplicationRecord
   has_many :schedules, dependent: :destroy
   accepts_nested_attributes_for :schedules
 
+  GENDERS = ['female', 'male'].freeze
```

Comment:
The problem with having this in a helper is paradigm: MVC and this is Sooo model.
It could be placed into a model concern and included into the models needed. But having such in helpers is just counter Paradigm. If it comes from a helper it is also insecure, because proper validation in the model is at least unpleasant.

But putting such into a concern would fit your quest, just not inside a helper.